diffpy/snmf/stretchednmfapp.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import argparse
+from pathlib import Path
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(
+        prog="stretched_nmf",
+        description="Stretched Nonnegative Matrix Factorization"
+    )
+    parser.add_argument('-i', '--input-directory', type=str, default=None,
+                        help="Directory containing experimental data. Defaults to current working directory.")
+    parser.add_argument('-o', '--output-directory', type=str,
+                        help="The directory where the results will be written. Defaults to '<input_directory/snmf_results>'.")
+    parser.add_argument('t', '--data-type', type=str, choices=['powder_diffraction', 'pdf'],
```

Comment:
when we write this we often make a global constant at the top of the file
`ALLOWED_DATA_TYPES = ['powder_diffraction', 'pd', 'pair_distribution_function', 'pdf']` then use an f-string in the help that lists the allowed choices (they may list by default in which case we don't need that...you can check.)